src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
-// write code here
+const employees = document.querySelector('ul');
+
+const stringToNumberConvertor = string => (
```

Comment:
Назва функції має починатися з дієслова
```suggestion
const convertStringToNumber= string => (
```